dlt/pipeline/pipeline.py

```diff
@@ -1114,38 +1117,29 @@ def _optional_sql_job_client(self, schema_name: str) -> Optional[SqlJobClientBas
             logger.info("Client not available due to missing credentials")
         return None
 
-    def _restore_state_from_destination(self, raise_on_connection_error: bool = True) -> Optional[TPipelineState]:
+    def _restore_state_from_destination(self) -> Optional[TPipelineState]:
         # if state is not present locally, take the state from the destination
         dataset_name = self.dataset_name
         use_single_dataset = self.config.use_single_dataset
         try:
             # force the main dataset to be used
             self.config.use_single_dataset = True
-            job_client = self._optional_sql_job_client(normalize_schema_name(self.pipeline_name))
-            if job_client:
-                # handle open connection exception silently
-                state = None
-                try:
-                    job_client.sql_client.open_connection()
-                except Exception:
-                    # pass on connection errors
-                    if raise_on_connection_error:
-                        raise
-                    pass
-                else:
-                    # try too get state from destination
-                    state = load_state_from_destination(self.pipeline_name, job_client.sql_client)
-                finally:
-                    job_client.sql_client.close_connection()
-
-                if state is None:
-                    logger.info(f"The state was not found in the destination {self.destination.__name__}:{dataset_name}")
+            schema_name = normalize_schema_name(self.pipeline_name)
+            with self._maybe_destination_capabilities():
+                schema = Schema(schema_name)
+                schema.update_normalizers()
+                schema._schema_name = schema.naming.normalize_identifier(schema_name)
```

Comment:
remove it and see, the tests will fail because the wrong not normalised schema name is used. maybe I am doing something wrong here, but the normalisers are not properly set in the schema if I keep it the way it was before.

Reply:
I'd say tests are wrong ie. comparing dataset name without normalizing. OK I'll take a look